theme/static/js/create-resource.js

```diff
@@ -21,95 +143,132 @@ $(document).ready(function() {
             $("#file-move-warning").hide();
         }
     });
-    $('#resource-type').on('change', function(){
-        $('#select-file').value = '';
-        $('#select-file').attr('value', '');
-        $('#irods-sel-file').text("No file selected.");
```

Comment:
@mauriel133 I think this resource-type change call back for clearing selected files, at least, for clearing out irods-sel-file is still needed. For example, if a user selected a ppt file for generic resource type, then the user selected a geo-raster resource type, the previous selection, i.e., ppt file, should be deselected automatically since the previously selected file does not apply to the new resource type. That was the previous behavior, and appears to be lost in the new revamped create resource implementation. Can you test to make sure that behavior still applies in the new implementation?